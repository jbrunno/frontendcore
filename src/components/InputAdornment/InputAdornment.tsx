import React from 'react';
import {
  InputAdornment as MuiInputAdornment,
  InputAdornmentProps as MuiInputAdornmentProps,
} from '@mui/material';

export type InputAdornmentProps = MuiInputAdornmentProps;

export function InputAdornment(props: InputAdornmentProps) {
  return <MuiInputAdornment {...props} />;
}

export default InputAdornment;
