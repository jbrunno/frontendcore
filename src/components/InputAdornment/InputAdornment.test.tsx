import React from 'react';
import { render } from '@test-utils/component-testing';
import { InputAdornment } from './InputAdornment';

test('renders the InputAdornment', () => {
  const { container } = render(<InputAdornment position="start" />);
  expect(container).toMatchSnapshot();
});
