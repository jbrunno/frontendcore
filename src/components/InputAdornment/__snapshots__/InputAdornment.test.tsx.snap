// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the InputAdornment 1`] = `
.emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  height: 0.01em;
  max-height: 2em;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  white-space: nowrap;
  color: #0000008A;
  margin-right: 8px;
}

<div>
  <div
    class="MuiInputAdornment-root MuiInputAdornment-positionStart emotion-0"
  >
    <span
      class="notranslate"
    >
      â€‹
    </span>
  </div>
</div>
`;
